packages/components/col/src/col.ts

```diff
@@ -76,22 +78,22 @@ export default defineComponent({
       pos.forEach((prop) => {
         const size = props[prop]
         if (typeof size === 'number') {
-          if (prop === 'span') classes.push(`el-col-${props[prop]}`)
-          else if (size > 0) classes.push(`el-col-${prop}-${props[prop]}`)
+          if (prop === 'span') classes.push(ns.b(`${props[prop]}`))
```

Comment:
![image](https://user-images.githubusercontent.com/39730999/152911068-33e316c6-bcdc-4016-8251-d285707790cd.png)

Because h is a number type, template strings are used here, otherwise the input parameter type of the `b` function needs to be modified